db/migrate/20210818083407_fix_setting_category.rb

```diff
@@ -0,0 +1,5 @@
+class FixSettingCategory < ActiveRecord::Migration[6.0]
```

Comment:
Can we have more specific name? this is clashing with the REX migration and Rails don't play well if two migrations have the same name.
```suggestion
class AnsibleSettingCategoryToDsl < ActiveRecord::Migration[6.0]
```